.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           python-version: '3.11'
 
       - uses: actions/download-artifact@v3
-        # Restore the rendered areifact to a directory outside the repo checkout.
+        # Restore the rendered artifact to a directory outside the repo checkout.
```

Comment:
Tiny bit of scope creep here just to fix a typo.
I am going with the en-US spelling "artifact" here (in preference to the en-GB spelling "artefact") as that is the spelling GitHub uses.